rosbag2_transport/src/rosbag2_transport/rosbag2_transport_python.cpp

```diff
@@ -170,6 +173,8 @@ static PyMethodDef rosbag2_transport_methods[] = {
   },
   {nullptr, nullptr, 0, nullptr}  /* sentinel */
 };
+# pragma GCC diagnostic pop
+#endif
```

Comment:
The inner block should still be used on non-Windows platforms, no? Currently the `ifdef` wraps all of it - you probably want to `endif` after line 160 and `ifdef` again before line 176.

Reply:
absolutely! Corrected in https://github.com/ros2/rosbag2/pull/116/commits/928a4eeea7874107accd1026ee1dd86601acd049